src/app/component/user/components/profile/shopping-list/shopping-list.component.scss

```diff
@@ -52,6 +56,18 @@ h2 > span {
     float: right;
 }
 
+
+.noItems {
+  font-family: $title-font;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 14px;
+  line-height: 17px;
+  margin-top: 30px;
+  text-align: center;
+  color: #bebebe;
```

Comment:
Colors should have the same writing style.

Reply:
done